packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutBase.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import { Responsive, WidthProvider, Layout, Layouts } from 'react-grid-layout';
+import {
+  IDashboardGridLayoutProps,
+  IDashboardGridLayoutStyles,
+  IDashboardCardLayout,
+  DashboardGridBreakpointLayouts
+} from './DashboardGridLayout.types';
+import { getStyles } from './DashboardGridLayout.styles';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { CardSizeToWidthHeight } from '../../utilities/DashboardGridLayoutUtils';
+
+require('style-loader!css-loader!react-grid-layout/css/styles.css');
+require('style-loader!css-loader!react-resizable/css/styles.css');
+require('style-loader!css-loader!./DashboardGridLayout.css');
+
+const ResponsiveReactGridLayout = WidthProvider(Responsive);
+
+export interface IDashboardGridLayoutBaseProps extends IDashboardGridLayoutProps {
+  createRGLLayouts?(dashBoardLayout: DashboardGridBreakpointLayouts): Layouts;
+  onLayoutChange?(currentLayout: Layout[], allLayouts: Layouts): void;
+}
+
+export class DashboardGridLayoutBase extends React.Component<IDashboardGridLayoutBaseProps, {}> {
+  /** The default props used for React-Grid-Layout */
+  public static defaultProps: Partial<IDashboardGridLayoutBaseProps> = {
+    rowHeight: 50,
```

Comment:
>rowHeight: 50, [](start = 4, length = 14)

@samiyaakhtar  can you please take a look at this number?

Reply:
Yes, we can either change it here or pass a prop from M365.

Reply:
Talked offline and I am making this 56 

---
In reply to: [216083082](https://github.com/OfficeDev/office-ui-fabric-react/pull/6285#discussion_r216083082) [](ancestors = 216083082)